src/components/molecules/about.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+	<div class="about">
+		<aboutTitle/>
+		<aboutSlogan/>
```

Comment:
I think common practice is to either write component tags LikeThis or like-this, not in camelCase. please check this issue